### ISSUE TITLE
Suggestion: Check if all Captions exists

### ISSUE THREAD

**Issue by DanielGoehler** [markbrummel/vscode-allint#15]
It would be very useful if missing Captions were shown as warning. (E.g. Setting could be allint.requiredCaptionsLanguages = "DEU,ENU,FRA")

**Comment by mrabus** [comment]
Given the changes in October preview [(Link)](https://blogs.msdn.microsoft.com/nav/2017/10/12/nav-development-preview-october-update/) and announcement of further changes I would propose to wait for the November preview

**Comment by markbrummel** [comment]
I agree and I knew this was in the works, but could not mention it yet. NDA and stuff. Sorry.
